app/models/audit_trail.rb

```diff
@@ -0,0 +1,3 @@
+class AuditTrail
```

Comment:
Do you have any sage advice about testing this class? I'll probably write some unit tests for it over the weekend.